cypress/e2e/marc-authority/verify-headingType-facet-option.cy.js

```diff
@@ -0,0 +1,70 @@
+import getRandomPostfix from '../../support/utils/stringTools';
+import TestTypes from '../../support/dictionary/testTypes';
+import DevTeams from '../../support/dictionary/devTeams';
+import Permissions from '../../support/dictionary/permissions';
+import TopMenu from '../../support/fragments/topMenu';
+import DataImport from '../../support/fragments/data_import/dataImport';
+import MarcAuthority from '../../support/fragments/marcAuthority/marcAuthority';
+import Users from '../../support/fragments/users/users';
+import JobProfiles from '../../support/fragments/data_import/job_profiles/jobProfiles';
+import MarcAuthorities from '../../support/fragments/marcAuthority/marcAuthorities';
+import Logs from '../../support/fragments/data_import/logs/logs';
+import MarcAuthorityBrowse from '../../support/fragments/marcAuthority/MarcAuthorityBrowse';
+
+describe('Importing MARC Authority files', () => {
+  const testData = {};
+  const jobProfileToRun = 'Default - Create SRS MARC Authority';
+  const fileName = '100_MARC_authority_records.mrc';
```

Comment:
where is this file?

Reply:
it`s already in master (from other ticket)